kontrollsamtale/infrastructure/src/main/kotlin/no/nav/su/se/bakover/kontrollsamtale/infrastructure/web/OpprettKontrollsamtaleRoute.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.su.se.bakover.kontrollsamtale.infrastructure.web
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.call
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.delete
+import io.ktor.server.routing.post
+import no.nav.su.se.bakover.common.brukerrolle.Brukerrolle
+import no.nav.su.se.bakover.common.infrastructure.web.Feilresponser
+import no.nav.su.se.bakover.common.infrastructure.web.Resultat
+import no.nav.su.se.bakover.common.infrastructure.web.authorize
+import no.nav.su.se.bakover.common.infrastructure.web.suUserContext
+import no.nav.su.se.bakover.common.infrastructure.web.svar
+import no.nav.su.se.bakover.common.infrastructure.web.withKontrollsamtaleId
+import no.nav.su.se.bakover.common.infrastructure.web.withSakId
+import no.nav.su.se.bakover.kontrollsamtale.domain.KontrollsamtaleService
+import no.nav.su.se.bakover.kontrollsamtale.domain.annuller.KunneIkkeAnnullereKontrollsamtale
+import no.nav.su.se.bakover.kontrollsamtale.domain.opprett.OpprettKontrollsamtaleCommand
+
+fun Route.opprettKontrollsamtaleRoute(
+    kontrollsamtaleService: KontrollsamtaleService,
+) {
+    post("/saker/{sakId}/kontrollsamtaler") {
+        authorize(Brukerrolle.Saksbehandler) {
+            call.withSakId { sakId ->
+                val opprettKontrollsamtaleCommand = OpprettKontrollsamtaleCommand(
+                    sakId = sakId,
+                    saksbehandler = call.suUserContext.saksbehandler,
+                )
```

Comment:
må ta inn ny dato for kontrollsamtalen

Reply:
Tar inn en innkallingsmåned. Fristen finner vi basert på det.